.readthedocs.yaml

```diff
@@ -7,6 +7,10 @@ sphinx:
   configuration: doc/conf.py
 
 python:
-  version: 3.8
   install:
     - requirements: doc/requirements.txt
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.8"
```

Comment:
`python.version` has been deprecated and should now be specified in the `build` tag